app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,156 @@
+@import "bootstrap";
+@import "font-awesome";
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+ $favorite-color: #355c7d;
+
+body{
+    background-color: $favorite-color;
```

Comment:
Please change this from Space: 2 to Space: 4.
Any file that is not **.html.erb** should be Spaces: 2.
![image](https://user-images.githubusercontent.com/35232348/95713270-d1d6a700-0c98-11eb-80e3-ca85894d4dcf.png)

Reply:
is this mine? im confused